extensions/objects/templates/math-expression-content-editor.component.ts

```diff
@@ -72,16 +72,18 @@ export class MathExpressionContentEditorComponent implements OnInit {
     // that an existing math expression is being edited. In this case, the
     // editor template can be initialised with the actual values instead of
     // default ones.
-    if (this.value.svg_filename && this.value.raw_latex) {
-      this.localValue.label = this.value.raw_latex;
-      this.value.mathExpressionSvgIsBeingProcessed = false;
-      this.convertLatexStringToSvg(this.localValue.label);
-    } else {
-      this.value.mathExpressionSvgIsBeingProcessed = true;
-    }
-    this.valueChanged.emit(this.value);
-    if (this.value && this.value.raw_latex) {
-      this.localValue.label = this.value.raw_latex;
+    if (this.value) {
```

Comment:
I don't think it can be. The value here can be null or undefined or an Object. I don't think there can be a case where a falsy value can be valid value here.

Reply:
So then why do you need this outer "if" clause at all?

Reply:
Because the value can be ` undefined or an Object`. I don't want this code running when the value is undefined?

Reply:
Null or undefined is a falsy value, isn't it?

In such a case, what _should_ happen?

Reply:
Nothing. If you closely look at the previous check, we had `if (this.value.svg_filename && this.value.raw_latex)`. (Just want to point our that we  still have it). I added an extra check to make sure that we don't get `can't get read svg_filename of undefined` kind of errors.

Reply:
Fixed it with the initialisation of this.value in ngOnInit.